onboarding/sprint-rotation.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Sprint Rotation
+description: What is the sprint rotation and how does it work?
+---
+
+## Sprint Rotation
+
+After an engineer has gone through basic onboarding (setting up systems, etc.) which will last about a week, they may spend a few sprints switching between different teams.
+
+### Why a sprint rotation?
+
+When engineers begin at Artsy, we want them to develop an understanding of our systems and business at a pace that fits their current experience. There's a lot of information to absorb and when engineers start on a product team with immediate sprint goals, there's often not the time or space to learn about the many facets of Artsy.
+
+The sprint rotation is an attempt to mitigate that worry, and also make onboarding/mentorship a shared responsibility. If an engineer switches teams at the start of a new quarter, they should be able to consume the same documentation or expertise that a team has developed for the sprint rotation.
+
+#### Goals
+- Get up-to-speed on Artsy's systems by being part of different product teams.
+- Learn about different sides of Artsy's business in a first-hand way.
+- Get to know our engineering team and how different engineers and teams work
+
+### How does the sprint rotation work?
+
+A new engineer's manager will work with tech leads to determine the proper length of the sprint rotation and which teams will be involved. This process can be used for the sole purpose of onboarding, and/or to help choose which team the new engineer will start on full-time.
+
+Following the sprint rotation, the new engineer will join a team full-time.
+
+#### Before the rotation starts
+- Manager schedules the duration and content of the sprint rotation, being cognizant of factors like:
+  - Other team members also going through sprint rotation
+  - Schedule of tech leads/PMs
+  - Specific areas of interest for new engineer
+
+#### Before the sprint starts
+_Manager responsibilities:_
+- Make sure the new engineer has met with the tech lead of the team for the upcoming sprint. This may have happened already during the first week.
+- Make sure the tech lead/PM are aware (well in advance) of the upcoming sprint rotation.
+
+_New engineer responsibilities:_
+- Engineer attends sprint planning for the team they'll be rotating onto.
+- Pending time, the engineer can get their dev environment set up with any additional systems.
+- Pending time, the engineer can browse the team's documentation and current sprint projects.
+
+_Tech lead/PM responsibilities:_
+- Add the engineer to all team meetings _just for that sprint_. This includes standups, knowledge shares, retros, and the sprint planning for the assigned sprint (which will likely occur the week before).
```

Comment:
I also added the new engineer to the @sell-pal user group to get full team experience. Not sure if it's worth mentioning.